src/FixedDataTableRow.js

```diff
@@ -216,7 +216,8 @@ class FixedDataTableRowImpl extends React.Component {
       'public/fixedDataTableRow/odd': this.props.index % 2 === 1,
       'public/fixedDataTableRow/even': this.props.index % 2 === 0,
     });
-    var fixedColumnsWidth = sumPropWidths(this.props.fixedColumns);
+    var fixedColumns=this.props.fixedColumns;
+    var fixedColumnsWidth = fixedColumns.length && (fixedColumns[fixedColumns.length-1].offset+fixedColumns[fixedColumns.length-1].props.width);
```

Comment:
Aren't we already calculating widths/offsets from the reducer? Let's make use of them here.